virtualizarr/xarray.py

```diff
@@ -41,9 +41,52 @@ def open_virtual_dataset(
     drop_variables: Iterable[str] | None = None,
     loadable_variables: Iterable[str] | None = None,
     decode_times: bool | None = None,
-    cftime_variables: Iterable[str] | None = None,
     indexes: Mapping[str, xr.Index] | None = None,
 ) -> xr.Dataset:
+    """
+    Open a data file, virtual dataset, Zarr store, etc. as an [xarray.Dataset][] wrapping virtualized zarr arrays.
```

Comment:
Maybe this? I definitely wouldn't say "open a virtual dataset ... as a xr.Dataset".

```suggestion
    Open an archival data source as an [xarray.Dataset][] wrapping virtualized zarr arrays.
```